rust/Cargo.toml

```diff
@@ -83,10 +83,11 @@ kanal = { version = "0.1.0-pre8", features = ["async"] }
 once_cell = "1.10.0"
 num_cpus = "1.16.0"
 pbjson = "0.5.1"
-prometheus = { version = "0.13.0", default-features = false }
+prometheus = { version = "0.13.3", default-features = false }
 prost = { version = "0.12.3", features = ["no-recursion-limit"] }
 prost-types = "0.12.3"
-rayon = "1.10.0"
+# Keep it compatible with the aptos-core version.
+rayon = "1.5.2"
```

Comment:
Hopefully nothing breaks with this change lol, 1.10 -> 1.5.2 seems like a big change

Reply:
this is the version aptos-core uses and here it's introduced by @grao1991 

let me know if you have any concern. I feel upgrade rayon from 1.5.2 to 1.10 at aptos-core side is more aggressive 🤦

Reply:
Going to merge and we can revisit this if there is performance degradation.